schemas/witnet/witnet.proto

```diff
@@ -286,7 +291,7 @@ message VrfProof {
 }
 
 message DataRequestVrfMessage {
-    CheckpointBeacon beacon = 1;
+    CheckpointVRF vrf_input = 1;
     Hash dr_hash = 2;
 }
```

Comment:
Maybe I'm missing something, but why are we putting `DataRequestVrfMessage` and `CheckpointVRF` into the schema if these structures are not being ever sent over the wire?

Reply:
Data Request VRF message was in the schema before this PR so I am assuming is somehow being sent right?

Reply:
We use the schema to define how to serialize this structure, because it needs to be serialized to bytes in order to create a VRF proof.

Reply:
Got it, sorry for my blatant ignorance :sweat_smile: